src/App.js

```diff
@@ -1,16 +1,27 @@
 import './App.css';
-import {Router , Route} from "react-router-dom"
+import {Routes, Route} from "react-router-dom"
+import {useState, useEffect} from 'react';
+import SignUp from './Pages/SignUp';
+import LogIn from './Pages/LogIn';
+import Dashboard from './Pages/Dashboard'
 
-function App() {
-  return (
-    <div>
-      <header>
-      </header>
-      <main>
-      <p> code works </p>
-      </main>
-    </div>
-  );
-}
+export default function App() {
+
+    const [assets, setAssets] = useState([])
```

Comment:
If `assets` are only going to be available in the dashboard it would be appropriate to move it's state into the Dashboard component.

Reply:
i want to store different data to each asset and plan on sending it through props accordingly where needed